lib/data/rates/state_pension.yml

```diff
@@ -35,6 +35,11 @@
   lower_weekly_rate: 75.50
 
 - start_date: 2019-04-08
-  end_date: 2020-04-08
+  end_date: 2020-04-05
   weekly_rate: 129.20
   lower_weekly_rate: 77.45
+
+- start_date: 2020-04-06
+  end_date: 2100-04-05
```

Comment:
2100?

Reply:
this is used in a couple of other rate files to make sure things don't stop working next year should things not be updated
https://github.com/alphagov/smart-answers/search?q=2100&unscoped_q=2100